languages/csharp/exercises/concept/switch-statements/SwitchStatements.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+
+// **** please do not modify the Manager class ****
+public class Manager
+{
+    public string Name { get; }
+    public string Activity { get; }
+
+    public Manager(string name, string activity)
+    {
+        this.Name = name;
+        this.Activity = activity;
+    }
+}
+
+// **** please do not modify the Incident enum ****
+public enum Incident
+{
+    RedCard,
+    YellowCard,
+    Foul,
+    Injury
+}
+
+public static class PlayAnalyzer
```

Comment:
Maybe move this class to the top to make it more clear to students where to start implementing? This will be especially useful in-browser I feel, where there is less real-estate.